packages/wdio-sauce-service/src/service.ts

```diff
@@ -109,6 +109,17 @@ export default class SauceService implements WebdriverIO.ServiceInstance {
     }
 
     afterTest (test: any, context: any, results: any) {
+        /**
+         * If the test failed push the stack to Sauce Labs in separate lines
+         * This should not be done for UP because it's not supported yet and
+         * should be removed when UP supports `sauce:context`
+         */
+        const { error } = results
+        if (error && !this._isUP){
+            const lines = error.stack.split(/\r?\n/)
```

Comment:
I wonder if we should limit the amount of lines to like 5 to ensure we don't run crazy many context calls. Also what would happen if we pass in HTML tags? Are they getting interpreted?

```suggestion
            const lines = error.stack.split(/\r?\n/).slice(0, MAX_ERROR_STACK_LENGTH)
```

Let's define `MAX_ERROR_STACK_LENGTH` somewhere at the top of the file.